components/dot_menu/__snapshots__/dot_menu.test.tsx.snap

```diff
@@ -227,14 +224,9 @@ exports[`components/dot_menu/DotMenu should match snapshot, on Center 1`] = `
     <MenuItemAction
       id="pin_post_post_id_1"
       onClick={[Function]}
-      show={true}
+      show={false}
       text="Pin"
     />
-    <li
-      className="MenuItem__divider"
-      id="divider_post_post_id_1_edit"
-      role="menuitem"
-    />
```

Comment:
Why does this get removed from the snapshot with just a typescript change?

Reply:
done in latest commit